Source/ThirdParty/RuntimeGifLibrary/RuntimeGifLibrary.Build.cs

```diff
@@ -0,0 +1,38 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+using System.IO;
+using UnrealBuildTool;
+
+public class RuntimeGifLibrary : ModuleRules
+{
+	public RuntimeGifLibrary(ReadOnlyTargetRules Target) : base(Target)
+	{
+		Type = ModuleType.External;
+
+		if (Target.Platform == UnrealTargetPlatform.Win64 || Target.Platform == UnrealTargetPlatform.Android)
+		{
+			PublicDefinitions.Add("WITH_LIBNSGIF=1");
+
+			string IncPath = Path.Combine(ModuleDirectory, "include");
+			PublicSystemIncludePaths.Add(IncPath);
+
+			PublicIncludePaths.Add(Path.Combine(ModuleDirectory, "include"));
+
+			// Add the import library
+			PublicAdditionalLibraries.Add(Path.Combine(ModuleDirectory, "x64", "Release", "libnsgif.lib"));
```

Comment:
If you bring back .lib file then why don't we use direct Linking agains DLL functions so that we reduce amount of duplicated code (i.e. DLLExport???)

Reply:
Ah my bad I just thought you want it back because as per our earlier discussion on this point I removed it but here I found a comment on it in review I pushed it back 😕

Reply:
Working on it